src/errors.js

```diff
@@ -11,6 +11,18 @@ class AbortError extends Error {
   }
 }
 
+class AllListenersFailedError extends Error {
+  constructor () {
+    super('AllListenersFailedError')
```

Comment:
This is the message right? Shouldn't this be more human readable?

Reply:
I made the error more verbose to try and provide an actionable error for users.